recipes/runit_systemd.rb

```diff
@@ -0,0 +1,28 @@
+#
+# Copyright:: Copyright (c) 2015 Opscode, Inc.
+#
+# All Rights Reserved
+#
+
+project_name = node['enterprise']['name'].clone
+project_name.gsub!(/_/, '-') if project_name == 'private_chef'
+unit_name = "#{project_name}-runsvdir-start.service"
+
+template "/usr/lib/systemd/system/#{unit_name}" do
+  owner "root"
+  group "root"
+  mode "0644"
+  variables({
+              :install_path => node[node['enterprise']['name']]['install_path'],
+              :project_name => project_name
+  })
+  source "runsvdir-start.service.erb"
+end
+
+execute "systemctl enable #{unit_name}" do
+  not_if "systemctl is-enabled #{unit_name}"
+end
+
+execute "systemctl start #{unit_name}" do
+  not_if "systemctl is-active #{unit_name}"
+end
```

Comment:
As I mentioned earlier, `service` is not used in the other two recipes so I didn't use it here. Any idea why that is?